intera_interface/src/intera_interface/limb.py

```diff
@@ -88,8 +102,9 @@ def __init__(self, limb="right", synchronous_pub=False):
         self._cartesian_pose = dict()
         self._cartesian_velocity = dict()
         self._cartesian_effort = dict()
-        self._joint_names = { limb: joint_names }
```

Comment:
I didn't find any examples of where this was being used. 

Also, from an encapsulation point of view, the Limb() object should only contain names for that limb.